src/modules/uORB/uORBDeviceMaster.cpp

```diff
@@ -395,15 +395,16 @@ void uORB::DeviceMaster::showTop(char **topic_filter, int num_filters)
 
 			PX4_INFO_RAW("\033[H"); // move cursor home and clear screen
 			PX4_INFO_RAW(CLEAR_LINE "update: 1s, num topics: %i\n", num_topics);
-			PX4_INFO_RAW(CLEAR_LINE "%-*s INST #SUB #MSG #LOST #QSIZE\n", (int)max_topic_name_length - 2, "TOPIC NAME");
+			PX4_INFO_RAW(CLEAR_LINE "%-*s INST #SUB #PRIO #MSG #LOST #QSIZE\n", (int)max_topic_name_length - 2, "TOPIC NAME");
 			cur_node = first_node;
 
 			while (cur_node) {
 
 				if (!print_active_only || cur_node->pub_msg_delta > 0) {
-					PX4_INFO_RAW(CLEAR_LINE "%-*s %2i %4i %4i %5i %i\n", (int)max_topic_name_length,
+					PX4_INFO_RAW(CLEAR_LINE "%-*s %2i %5i %4i %4i %5i %i\n", (int)max_topic_name_length,
```

Comment:
5 digits for orb priority? 

I'm actually wondering if this change should be omitted. The ORB priority is really only metadata used by the sensors module, and even that should probably be rethought. Displaying it might be more misleading than it's worth.

Reply:
That was either a lazy copy/paste or I was trying to center the columns better under the " PRIO" heading... it's been a while.

That's a fair point and I'm happy to omit it. I find it useful for verifying priorities for sensor additions, but agree it's probably misleading for more. I'll remove it.